### PR TITLE
[Impeller] migrate more AIKS test to DL.

### PR DESCRIPTION
**Pull request by jonahwilliams**
Part of https://github.com/flutter/flutter/issues/142054

### CODE REVIEW

**Comment by jonahwilliams** [comment]
Leaving golden diffs untriaged, its just a minor difference in the color values for the gradient.

File: impeller/display_list/aiks_dl_basic_unittests.cc [hunk 1]
```diff
@@ -762,5 +775,290 @@ TEST_P(AiksTest, SolidColorCirclesOvalsRRectsMaskBlurCorrectly) {
...
+      1.0,
+  };
+  DlPaint paint;
+  paint.setColorSource(DlColorSource::MakeLinear({0, 0},              //
+                                                 {100, 100},          //
+                                                 2,                   //
+                                                 colors.data(),       //
+                                                 stops.data(),        //
+                                                 DlTileMode::kRepeat  //
+                                                 ));
```
Comment (gaaclarke) [inline impeller/display_list/aiks_dl_basic_unittests.cc:797]: Argument comments would help for the literals here.

File: impeller/display_list/aiks_dl_basic_unittests.cc [hunk 2]
```diff
@@ -762,5 +775,290 @@ TEST_P(AiksTest, SolidColorCirclesOvalsRRectsMaskBlurCorrectly) {
...
+
+  SkRect clip_rect = SkRect::MakeLTRB(50, 50, 400, 300);
+  SkRRect clip_rrect = SkRRect::MakeRectXY(clip_rect, 100, 100);
+
+  // Draw a clipped SaveLayer, where the clip coverage and SaveLayer size are
+  // the same.
+  builder.ClipRRect(clip_rrect, DlCanvas::ClipOp::kIntersect);
+
+  DlPaint save_paint;
+  auto bdf = std::make_shared<DlColorFilterImageFilter>(
```
[the remark this answers is not in the document]
Comment (jonahwilliams) [reply impeller/display_list/aiks_dl_basic_unittests.cc]: Done

File: impeller/display_list/aiks_dl_basic_unittests.cc (hunk 1 above)
Comment (jonahwilliams) [reply impeller/display_list/aiks_dl_basic_unittests.cc]: Done

**Pull request merged**